### PR TITLE
Create full mobile version

### PR DESCRIPTION
**Pull request by Laylooo**
In index.html:

- added web-pages attributes;
- added about me page;

In styles.css :

- created style for new webpages;
- used flexbox and other thing which I have learned from lessons;

Added images and some SVG files.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -56,7 +56,159 @@ <h6 class="social-media-text">LET’S CONNECT</h6>
 
 				</ul>
 			</section>
-		</main>
+		</section>
+		<section class="works-section">
```
Comment (rloterh) [inline index.html:60]: - [ ] To ensure a proportional alignment, lets ensure that the main container of this section uses the given margins defined in the Figma design template, **top, bottom, left, right**.

**Review by rloterh — CHANGES_REQUESTED** [review]
# Changes Required! ♻️

Hi @Laylooo 👋 👋, I'm Robert, your reviewer for this PR.

I must commend you for the nice work done👏.  You nailed almost all of the specifications of this project right. There some necessary changes we need to implement to make our application more robust.

## Good points 👍
No Linters errors ✔️
Good use of HTML semantics ✔️


## Aspects to improve ♻️

_Check the comment(s) under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You can also consider:**

- [OPTIONAL] The **Author** section of our ReadMe lacks enough info about the author(s). Let's improve this aspect by adding additional profile links such as LinkedIn, Twitter, and Github, and ensure that the highlighted profile has links that navigate to the corresponding profile URL when clicked. Please see the below example:

## Author

👤 **Robert Loterh**

- GitHub: [@rloterh](https://github.com/rloterh)
- Twitter: [@RLoterh](https://twitter.com/RLoterh)
- LinkedIn: [LinkedIn](https://linkedin.com/in/robert-loterh)


Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (ElijahTobs) [reply index.html:60]: Not addressed

File: index.html [hunk 2]
```diff
@@ -56,7 +56,159 @@ <h6 class="social-media-text">LET’S CONNECT</h6>
...
+					<ul class="tech-info">
+						<li>html</li>
+						<li>css</li>
+						<li>javaScript</li>
+					</ul>
+					<button class="btn" type="button">See Project</button>
+				</li>
+			</ul>
+		</section>
+		<section class="about">
```
[the remark this answers is not in the document]
Comment (ElijahTobs) [reply index.html:142]: Well addressed 💯

File: index.html (hunk 1 above)
Comment (ElijahTobs) [reply index.html:60]: Well addressed 💯

File: index.html [hunk 3]
```diff
@@ -56,7 +56,159 @@ <h6 class="social-media-text">LET’S CONNECT</h6>
...
+						</li>
+						<li>
+							<a href="https://www.google.com/">
+								<img class="icons" src="/images/social_media/Vector5.svg" alt="google">
+							</a>
+						</li>
+					</ul>
+				</section>
+
+				<button class="btn_1" type="button">Get my resume</button>
```
[the remark this answers is not in the document]
Comment (ElijahTobs) [reply index.html:176]: Not well addressed

File: index.html [hunk 4]
```diff
@@ -56,7 +56,159 @@ <h6 class="social-media-text">LET’S CONNECT</h6>
...
+						<li class="canopy1">2015</li>
+					</ul>
+					<p class="despription">A daily selection of privately personalized reads; no accounts or
+						sign-ups required.</p>
+					<ul class="tech-info">
+						<li>html</li>
+						<li>css</li>
+						<li>javaScript</li>
+					</ul>
+					<button class="btn" type="button">See Project</button>
```
[the remark this answers is not in the document]
Comment (ElijahTobs) [reply index.html:79]: Not well addressed

File: index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (Laylooo) [reply index.html:60]: can you clarify what I have to do?

[the remark this answers is not in the document]
Comment (ElijahTobs) [reply index.html:60]: Sure, I will. Give me some minutes to wrap up with the review.

File: index.html [hunk 5]
```diff
@@ -56,7 +56,159 @@ <h6 class="social-media-text">LET’S CONNECT</h6>
...
+				</li>
+			</ul>
+		</section>
```
Comment (ElijahTobs) [inline index.html:141]: - [ ] As the previous reviewer stated, the main container (`<section class="works-section">     </section>`) of this section I've highlighted should use a `display: grid;` property and value.

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required ⚠️ 🚦
## Hi @Laylooo 👋

You've done really well so far! 👍 

## Major Highlights 💯 

- Your README is professional and you've followed the Github flow rules. ✔️ 
- You have good commit messages ✔️ 
- All linter checks are passing ✔️ 

_However, there are some changes that you are required to make before this project can be approved._

## Required Changes ♻️




_See the inline comments below_

## Optional Changes 👌
_Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better._

Check for changes marked as Optional in the inline comments below.


_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

#### "Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."

### Note that you still have 1 more review as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561#), however, if you feel this review is unfair, you can request a second opinion using this [form](https://airtable.com/shrQAqnBwek5a0O0s).

## Happy Coding :computer: 

_Reviewed by_  **Elijah O. Ayandokun**

**Review by Shaher-11 — APPROVED** [review]
Hello @Laylooo ,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle: 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**